components/doner-card/index.js

```diff
@@ -0,0 +1,94 @@
+import { Grid, GridItem, Circle } from "@chakra-ui/layout";
+import { Box } from "@chakra-ui/react";
+import { PhoneIcon } from "@chakra-ui/icons";
+import { ImLocation } from "react-icons/im";
+
+// DonerCard Component can be resized by providing it with the width argument
+// other informations about the card can be provided to obtain reusability however we want.
+export default function DonerCard({
+  name = "Elyas Salar",
+  bloodType = "B+",
+  phoneNumber = "0772 106 8504",
+  city = "Chamchamal",
+  width = 300,
```

Comment:
Let's remove those default values, and remove the width property. Make the component responsive on Desktop, Table, and Mobile by default no need to provide any arguments.